### PR TITLE
update: follower fetching content updated to include GCP

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes
ref: https://aiven.atlassian.net/browse/HH-4231
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by MichaelTansiniAiven — APPROVED** [review]
no issues from me

**Review by gpkc — APPROVED** [review]

File: docs/products/kafka/concepts/follower-fetching.md [hunk 1]
```diff
@@ -15,7 +16,8 @@ import Followerfetching from "@site/static/images/content/figma/follower-fetchin
    receive data, improving overall performance.
 
 :::note
-Follower fetching is currently supported on AWS (Amazon Web Services).
+Follower fetching is supported on AWS (Amazon Web Services) and
+GCP (Google Cloud Platform).
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/concepts/follower-fetching.md:20]: They renamed GCP to Google Cloud a while back.
```suggestion
Google Cloud.
```

File: docs/products/kafka/concepts/follower-fetching.md [hunk 2]
```diff
@@ -46,9 +49,9 @@ high availability.
   the `broker.rack` setting, eliminating the need for manual configuration.
 
 - `client.rack`:  This setting on the Apache Kafka consumer indicates the AZ where
-  the consumer is running. It allows you to fetch data from the nearest replica. For
-  example, if your `client.rack` is set to `use1-az1`, it fetches data from an
-  Apache Kafka broker with `broker.rack` set to `use1-az1`.
+  the consumer is running. It allows you to fetch data from the nearest replica.
+  For example, setting `client.rack` to `use1-az1` on AWS or `europe-west1-b` on GCP
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/concepts/follower-fetching.md:53]:
```suggestion
  For example, setting `client.rack` to `use1-az1` on AWS or `europe-west1-b` on Google Cloud
```

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 3]
```diff
@@ -19,17 +19,23 @@ Enabling follower fetching in Aiven for Apache Kafka® allows your consumers to
 - [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs).
 
 :::note
-Follower fetching is currently supported on AWS (Amazon Web Services).
+Follower fetching is supported on AWS (Amazon Web Services) and
+GCP (Google Cloud Platform).
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:23]: Same as other comment

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 4]
```diff
@@ -19,17 +19,23 @@ Enabling follower fetching in Aiven for Apache Kafka® allows your consumers to
...
-physical location might have different AZ names in different accounts. To ensure
-consistency when configuring `client.rack`, use the AZ ID, which remains the same
-across accounts.
+- **AWS**: In AWS, availability zone (AZ) names can vary across different accounts.
+  The same physical location might have different AZ names in different accounts. To
+  ensure consistency when configuring `client.rack`, use the AZ ID, which remains the same
+  across accounts.
 
-To map AZ names to AZ IDs, see [AWS Knowledge Center article](https://repost.aws/knowledge-center/vpc-map-cross-account-availability-zones) and the [AWS documentation on AZ IDs](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html).
+  To map AZ names to AZ IDs, see
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:33]:
```suggestion
  To map AZ names to AZ IDs, see the
```

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 5]
```diff
@@ -19,17 +19,23 @@ Enabling follower fetching in Aiven for Apache Kafka® allows your consumers to
...
+  [AWS Knowledge Center article](https://repost.aws/knowledge-center/vpc-map-cross-account-availability-zones)
+  and the [AWS documentation on AZ IDs](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html).
+
+- **GCP**: In GCP, use the AZ name directly as the `client.rack` value.
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:37]: Same as other comments on GCP

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 6]
```diff
@@ -148,57 +154,64 @@ Parameters:
 ## Client-side configuration
 
 To enable follower fetching at the client level, configure the `client.rack` setting
-in the Apache Kafka client. Set the `client.rack` value to the corresponding AZ ID
-for each client. This ensures the client fetches data from the nearest replica.
+in the Apache Kafka client. Set the `client.rack` value to the corresponding AZ ID for
+AWS or AZ name for GCP for each client. This ensures the client fetches data from the
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:158]: Same for GCP here

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 7]
```diff
@@ -148,57 +154,64 @@ Parameters:
...
+nearest replica.
 
-Add this example configuration to your consumer properties file:
+Example configuration for your consumer properties file:
 
 ```plaintext
-client.rack=use1-az1
+client.rack=use1-az1 # AWS example
+client.rack=europe-west1-b # GCP example
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:165]:
```suggestion
client.rack=europe-west1-b # Google Cloud example
```

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 8]
```diff
@@ -148,57 +154,64 @@ Parameters:
...
 ```
 
 ### Example scenario: follower fetching in different AZs
 
 Assume you have an Aiven for Apache Kafka cluster running in two AZs in the `us-east-1`
-region. The AZ IDs are `use1-az1` and `use1-az2`. You also have consumers distributed
-across three AZs: `use1-az1`, `use1-az2`, and `use1-az3`.
-
-#### Cluster setup
+region for AWS and in the `europe-west1` region for GCP:
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:171]:
```suggestion
region for AWS and in the `europe-west1` region for Google Cloud:
```

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 9]
```diff
@@ -148,57 +154,64 @@ Parameters:
...
+#### Cluster setup and consumer distribution
 
-- Consumers are in:
-  - `use1-az1`
-  - `use1-az2`
-  - `use1-az3`
+| Cloud | Region         | AZs for brokers               | AZs for consumers                             |
+|-------|----------------|-------------------------------|-----------------------------------------------|
+| AWS   | `us-east-1`    | `use1-az1`, `use1-az2`        | `use1-az1`, `use1-az2`, `use1-az3`            |
+| GCP   | `europe-west1` | `europe-west1-b`, `europe-west1-c` | `europe-west1-b`, `europe-west1-c`, `europe-west1-d` |
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:178]:
```suggestion
| Google Cloud   | `europe-west1` | `europe-west1-b`, `europe-west1-c` | `europe-west1-b`, `europe-west1-c`, `europe-west1-d` |
```

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 10]
```diff
@@ -148,57 +154,64 @@ Parameters:
...
 
 #### Consumer configuration
 
-To configure consumers, set the `client.rack` value to the respective
-AZ ID for each consumer:
+Set the `client.rack` value to the respective AZ ID for AWS or AZ name for GCP for each consumer:
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:182]:
```suggestion
Set the `client.rack` value to the respective AZ ID for AWS or AZ name for Google Cloud for each consumer:
```

File: docs/products/kafka/howto/enable-follower-fetching.md [hunk 11]
```diff
@@ -148,57 +154,64 @@ Parameters:
...
 
-# For consumers in use1-az2
+# AWS consumers in use1-az2
 client.rack=use1-az2
 
-# For consumers in use1-az3
+# AWS consumers in use1-az3
 client.rack=use1-az3
+
+# GCP consumers in europe-west1-b
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/howto/enable-follower-fetching.md:194]: Same for all of these

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM. Just added comments about the name change for GCP.

**Pull request merged**